cove/templates/terms.html

```diff
@@ -1,217 +1,30 @@
 {% extends 'base.html' %}
 {% load i18n %}
 {% block content %}
+    {% with app=request.current_app %}
+        <div>
+            {# The content is split into many small templates.
+               This is so if one cove instance wants to change only one small bit of the page, they can do so by providing replacement templates of only the bit they want to change.
+               For example, a cove instance may want to keep most of the page the same but add legal content for another tracker.
+            #}
```

Comment:
Note these comment tags don't work in Django, instead use https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#comment

Reply:
Don't they? I did test it, wierd .... ok.

Reply:
fixed and tested again

Reply:
Weird. Are you testing with a later Django version?